### PR TITLE
OCM-10333 | ci: fix ids:57094,55883,37499

### PR DESCRIPTION
**Pull request by jameszwang**

### CODE REVIEW

File: tests/e2e/test_rosacli_upgrade.go [hunk 1]
```diff
@@ -483,7 +478,7 @@ var _ = Describe("Describe/List rosa upgrade",
 					By("Check list upgrade")
 					out, err := upgradeService.ListUpgrades(clusterID)
 					Expect(err).To(BeNil())
-					Expect(out.String()).To(ContainSubstring("%s   recommended - scheduled for %s %s UTC", upgradingVersion,
+					Expect(out.String()).To(ContainSubstring("%s  recommended - scheduled for %s %s UTC", upgradingVersion,
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_upgrade.go:481]: Better to use regex to match in case of future fixing.

Comment (jameszwang) [reply tests/e2e/test_rosacli_upgrade.go]: Updated PR according to comment.

File: tests/e2e/test_rosacli_upgrade.go [hunk 2]
```diff
@@ -483,8 +479,14 @@ var _ = Describe("Describe/List rosa upgrade",
 					By("Check list upgrade")
 					out, err := upgradeService.ListUpgrades(clusterID)
 					Expect(err).To(BeNil())
-					Expect(out.String()).To(ContainSubstring("%s   recommended - scheduled for %s %s UTC", upgradingVersion,
-						scheduledDate, scheduledTime))
+
+					pattern := `\d+\.\d+\.\d+\s+recommended\s+-\s+scheduled\s+for\s+\d{4}-\d{2}-\d{2}\s+\d{2}:\d{2}\s+UTC`
+					re := regexp.MustCompile(pattern)
+					matchedSubString := re.FindString(out.String())
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_upgrade.go:485]:
```suggestion
Expect(err).To(MatchRegexp(`%s\s+recommended\s+-\s+scheduled for %s %s UTC`, upgradingVersion,
				scheduledDate, scheduledTime))
```

Comment (jameszwang) [reply tests/e2e/test_rosacli_upgrade.go]: upgraded according to comments.

File: tests/e2e/test_rosacli_upgrade.go [hunk 3]
```diff
@@ -275,17 +276,24 @@ var _ = Describe("Cluster Upgrade testing",
 		It("to upgrade NON-STS rosa cluster across Y stream - [id:37499]", labels.Critical, labels.Runtime.Upgrade, func() {
 
 			By("Check the cluster version and compare with the profile to decide if skip this case")
+			if profile.Version != con.YStreamPreviousVersion || profile.ClusterConfig.STS {
+				Skip("Skip this case as the version defined in profile is not y-1 for non-sts cluster upgrading testing")
+			}
+
+			By("Check the cluster upgrade state to decide if skip this case")
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_upgrade.go:283]: Let's check the available upgrade versions of the cluster in case there is a bug in rosacli

**Pull request merged**